doctrine/doctrine-migrations-bundle/1.2/etc/packages/migrations.yaml

```diff
@@ -0,0 +1,5 @@
+doctrine_migrations:
+    dir_name: '%kernel.project_dir%/DoctrineMigrations'
```

Comment:
this should not be at the root. Migrations are code, and so should be in `src/`.

Btw, this means that the default value of `%kernel.root_dir%/DoctrineMigrations` will work fine for people putting their kernel in `src` (which is the case in Flex).